### PR TITLE
Tokenizer

### PR DESCRIPTION
**Pull request by taczin**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -84,11 +95,7 @@ def __init__(self, task_iters, method, custom_task_ratio=None):
             dataset_ids = [task_iter.dataset.id for task_iter in task_iters]
             task_ratio = [math.sqrt(task_iter.dataset.tensors[0].shape[0])/dataset_ids.count(task_iter.dataset.id) for task_iter in task_iters]
         else:
-            if custom_task_ratio == 'equal':
```
Comment (ChristophHoenes) [inline tasks.py:87]: Was this deleted on purpose? Or is this just to a mismatch between your branch and master? I believe this was a recent feature

File: util.py [hunk 2]
```diff
@@ -209,22 +209,23 @@ def parse_nonlinearity(nonlinearity_str):
 # FIXME: tasks are imported at this point to avoid a circular dependency:
 # tasks [*] -> models [SLClassifier] -> util [parse_nonlinearity]
 from tasks import *
+import tasks
 
 def get_task_by_name(args, task_name):
     if 'Offenseval' == task_name:
-        return OffensevalTask(cls_dim=args.mlp_dims[-1])
+        return tasks.OffensevalTask(cls_dim=args.mlp_dims[-1])
```
Comment (ChristophHoenes) [inline util.py:216]: Now we import tasks twice (from tasks import *, and import tasks). Are you sure that this way we do not have problems with circular imports/ Are there no other functions/classes of tasks used in the file?

File: tasks.py (hunk 1 above)
Comment (taczin) [reply tasks.py:87]: I think this was by accident. I'll change it back

File: util.py (hunk 2 above)
Comment (taczin) [reply util.py:216]: Originally it just said from tasks import * and it didn't work for me so I added the other import. Now it works. Hopefully it doesn't hurt ;)

Comment (baleato) [reply util.py:216]: It should work both ways, I might create another commit to remove the "from tasks import *" later.

**Review by baleato — APPROVED** [review]

**Pull request merged**